NFIQ2/NFIQ2Algorithm/src/prediction/RandomForestML.cpp

```diff
@@ -150,9 +150,37 @@ NFIQ::Prediction::RandomForestML::initModule(
 
 void
 NFIQ::Prediction::RandomForestML::evaluate(
-    const std::vector<NFIQ::QualityFeatureData> &featureVector,
+    std::unordered_map<std::string, NFIQ::QualityFeatureData> &featureMap,
     const double &utilityValue, double &qualityValue, double &deviation) const
 {
+	std::vector<std::string> rfFeatureOrder { "FDA_Bin10_0", "FDA_Bin10_1",
```

Comment:
Use the constants, lest these change. Can probably also make this vector `static const`, or even an `array` to save some space?